mod/deckorate_core/set/type/metric/answer_validation.rb

```diff
@@ -16,7 +16,8 @@ def validate_all_numeric_values
   bad_answer = metric_card.answers.find do |answer|
     !valid_numeric_value? answer.value
   end
-  "Non-numeric value: '#{bad_answer.value}'" if bad_answer.present?
+  return t(:answer_validation_error_message, bad_answer: bad_answer.value) \
+    if bad_answer.present?
```

Comment:
CodeClimate didn't object to this? The `return` is superfluous, and the backslash makes things hard to read.

more typical would be:

```
  ...
  return unless bad_answer.present?

  t :answer_validation_error_message, bad_answer: bad_answer.value
end
```